### PR TITLE
[MIG][10.0] migrate product_sequence

### PR DESCRIPTION
**Pull request by SodexisTeam**
Minor version change and renaming openerp to odoo imports

### CODE REVIEW

File: product_sequence/README.rst [hunk 1]
```diff
@@ -13,13 +18,20 @@ they already have a unique reference (or no reference) set.  Products with a def
...
+   :target: https://runbot.odoo-community.org/runbot/135/10.0
+
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/product-attribute/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_sequence%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`here <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_sequence%0Aversion:%2010.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: Use latest template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: product_sequence/data/product_sequence.xml [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-            <field name="code">product.product</field>
-        </record>
+<!--
+  product_sequence for Odoo
+  Copyright (C) 2004 Tiny SPRL
+  Copyright (C) 2016 Sodexis
+  The licence is in the file __openerp__.py
+-->
 
+<odoo>
```
Comment (pedrobaeza) [inline product_sequence/data/product_sequence.xml]: Use `<odoo noupdate="1"/>`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
2 things commented.

**Comment by atchuthan** [comment]
@pedrobaeza fixed your comments

**Review by pedrobaeza — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**